### PR TITLE
Add install instructions for the Moto 360 2015.

### PR DESCRIPTION
**Pull request by MagneFire**
Also update porting status page accordingly.

I've added a link to the https://github.com/AsteroidOS/meta-smelt-hybris/blob/master/README-system-dir.md#0-connect-the-watch-via-usb for instructions on how to connect the watch via USB.
Please only merge this PR if the builds for smelt are available.

### CODE REVIEW

File: pages/install/smelt.hbs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+</tr></tbody></table>
+
+<div class="callout callout-info">
+    <h4>USB connection</h4>
+    <p>The Moto 360 2015 does not expose a USB connection. <a href="https://github.com/AsteroidOS/meta-smelt-hybris/blob/master/README-system-dir.md#0-connect-the-watch-via-usb">You will need to solder a USB connector or create an adapter for it.</a></p>
```
Comment (FlorentRevest) [inline pages/install/smelt.hbs:32]: Can't you inline the content of that README in the webpage?

Comment (MagneFire) [reply pages/install/smelt.hbs:32]: Sure I can do that too. Do you want me to include the images as well?

File: pages/install/smelt.hbs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+</tr></tbody></table>
+
+<div class="callout callout-info">
+    <h4>USB connection</h4>
+    <p>The Moto 360 2015 doesn't expose a USB connection. It is however possible to connect to the watch using an adapter cable.</p>
+    <p>You will need access to the pads of the watch, this can be achieved by removing plastic outer ring. This is fairly straightforward as the ring can be easily lifted by putting a fingernail underneath the small lip.</p>
+    <p>Then you can either solder to the appropriate pads or 3D print a dock.
```
Comment (FlorentRevest) [inline pages/install/smelt.hbs:34]: isn't that missing a </p> ?

Comment (MagneFire) [reply pages/install/smelt.hbs]: wow, how did I miss that ;/

**Pull request merged**